presto-server/pom.xml

```diff
@@ -24,4 +23,48 @@
         <main-class>io.prestosql.server.PrestoServer</main-class>
         <process-name>${project.artifactId}</process-name>
     </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireFilesSize>
+                                    <maxsize>1073741824</maxsize>
+                                    <files>
+                                        <file>${project.build.directory}/${project.artifactId}-${project.version}.tar.gz</file>
```

Comment:
will the check fail if this file does not exist?

Reply:
yes.